filters/filters-2020.txt

```diff
@@ -2596,13 +2596,13 @@ kabel-tv-plus.blogspot.com##+js(acs, addEventListener, google_ad_client)
 videodotados.com##+js(acs, document.querySelectorAll, popMagic)
 
 ! shrinke.me anti adb popup
-shrinke.me##+js(aopr, app_vars.force_disable_adblock)
-shrinke.me##+js(aopr, open)
-shrinke.me##+js(set, blurred, false)
-shrinke.me###imgAddDirectLink
-||shrinke.me/sw.js$script,1p
-||shrinkme.io^$script,3p
-*$script,3p,denyallow=cloudflare.com|cloudflare.net|google.com|gstatic.com|hwcdn.net|jquery.com|recaptcha.net,domain=shrinke.me
+shrinke.*##+js(aopr, app_vars.force_disable_adblock)
+shrinke.*##+js(aopr, open)
+shrinke.*##+js(set, blurred, false)
+shrinke.*##.blog-item, .copyright-container, .col-md-offset-1.col-md-10 > center, .clickbtnt.expop, #cookie-pop, #imgAddDirectLink
```

Comment:
@BlazeFTL We don't explicitly block cookie popup for legal reason. I'll remove `#cookie-pop`.